docs/distributions/nixos/installation.md

```diff
@@ -185,15 +209,74 @@ If you would like to organize your configuration a little better, check out othe
 
 ## Wi-Fi and Bluetooth setup
 
+### Pre-Installation Steps
+
+Choose a method below and carefully read the notes. Afterwards, follow the [Wi-Fi and Bluetooth Guide on macOS](../../guides/wifi-bluetooth.md#setting-up) (firmware guide) and come back.
+
+=== ":fontawesome-solid-check: Method 1"
+    Method 1 requires you to run the firmware script twice, both in macOS and Linux. If you have uninstalled macOS, you may still be able to obtain the firmware files via a 600MB macOS recovery image without reinstalling macOS. See the linked guide for more information.
```

Comment:
You have already mentioned downloading of macOS recovery image from apple in method 5. Why repeating it?

Reply:
My bad - forgot to update that section. Fixed now.